src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +56,8 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+    //implement printable
+    public static Printable getPrintableString( Printable EXAMPLE) {
+		return EXAMPLE;
```

Comment:
It is incorrect to return the EXAMPLE. When you are creating an Address instance you are storing the respecting value in the variable 'value' in line 15. The assignment is happening in line 29. 

What is the point of passing in the EXAMPLE into the getPrintableString()?

I have seen you make this mistake repeatedly. So I encourage you to read the code closely with a teammate/friend and try to understand the logic behind it. If you need, I'm available for consultation. 

In addition, you were supposed to have a prefix in front of the value i.e. Adddress: 38, College Avenues